.travis.yml

```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```

Comment:
https://github.com/yarnpkg/yarn/issues/2445
non-interactive mode on `yarn publish` doesn't exist. I will revert this to npm publish for now unless anyone has concerns. cc @mattkime @mmcgahan